scripts/clearoutputs.py

```
@@ -0,0 +1,28 @@
+    print('Exiting...')
+    sys.exit(0)
+
+# Use glob to find all matching files
+files_to_remove = glob.glob('assets/*/output.json')
+files_to_remove += glob.glob('exceptions.log')
+
+for file_path in files_to_remove:
+    try:
+        os.remove(file_path)
+        print(f"Removed: {file_path}")
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)